backend/src/auth/auth.controller.ts

```diff
@@ -0,0 +1,60 @@
+import { Controller, Get, Req, Res, UseGuards } from '@nestjs/common';
+import { Request, Response } from 'express';
+import { AuthService } from './auth.service';
+import { GoogleGuard } from './guards/google.guard';
+import { JwtGuard } from './guards/jwt.guard';
+import { JwtRefreshGuard } from './guards/jwtrefresh.guard';
+
+@Controller()
+export class AuthController {
+  constructor(private AuthService: AuthService) {}
+
+  @Get('google')
+  @UseGuards(GoogleGuard)
+  async googleAuth() {}
+
+  @Get('google/redirect')
+  @UseGuards(GoogleGuard)
```

Comment:
When I log in and get redirected from the server, I attempt to refresh the same page, but I encounter an exception from Google Guard.